analyzer-core/src/main/java/ffb/analyzer/core/queue/PriorityTask.java

```diff
@@ -0,0 +1,32 @@
+package ffb.analyzer.core.queue;
+
+
+
+public class Task implements Comparable<Task> {
+
+    public Long id;
+    public int priority;
+
+
+    @Override
+    public int compareTo(Task otherTask) {
+        return this.getId().compareTo(otherTask.getId());
+    }
+
+
+    public int getPriority() {
+        return priority;
+    }
+
+    public void setPriority(int priority) {
```

Comment:
I don't think this should be exposed via a setter. This should only be set in the constructor.

Reply:
That's fair. Once we create a task with a given priority, we shouldn't really change that. Removed the setter.